sjk-core/src/test/java/org/gridkit/jvmtool/cmd/StackSampleAnalyzerCmdTest.java

```diff
@@ -0,0 +1,49 @@
+package org.gridkit.jvmtool.cmd;
+
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DateFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DecimalFormater;
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.MemRateFormater;
+import org.junit.Assert;
+import org.junit.Rule;
```

Comment:
Do we need this Rule import if we already import org.junit.rules.ExpectedException?

Reply:
It is needed for `thrown`.